packages/expo-image-manipulator/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### 🛠 Breaking changes
 
 - Bumped iOS deployment target to 13.4. ([#25063](https://github.com/expo/expo/pull/25063) by [@gabrieldonadel](https://github.com/gabrieldonadel))
+- Added `extent` action ([#25116](https://github.com/expo/expo/pull/25116) by [@LinusU](https://github.com/LinusU))
```

Comment:
It's not a breaking change, please move it to the correct section ;)

Reply:
@lukmccall Whoops 😅  fixed (and also rebased on latest main)